internal/engine/table.go

```diff
@@ -62,58 +53,30 @@ type Table struct {
 func (e Engine) LoadTable(name string) (table.Table, error) {
 	e.profiler.Enter("load table").Exit()
 
-	tablesPage, err := e.tablesPageContainer.Load()
+	tableMeta, err := e.dbfs.Table(name)
```

Comment:
`TableMeta` ? Isn't it the actual table?

Reply:
Nope, just an accessor to table files.